pallets/funding/src/lib.rs

```diff
@@ -18,96 +18,51 @@
-//! * [`ProjectsMetadata`]: Map of the assigned id, to the main information of a project.
-//! * [`ProjectsDetails`]: Map of a project id, to some additional information required for ensuring correctness of the protocol.
-//! * [`Bids`]: Double map linking a project-user to the bids they made.
-//! * [`Evaluations`]: Double map linking a project-user to the PLMC they bonded in the evaluation round.
-//! * [`Contributions`]: Double map linking a project-user to the contribution tokens they bought in the Community or Remainder round.
-//!
-//! ## Credentials
-//! The pallet will only allow users with certain credential types, to execute certain extrinsics.:
-//!
-//!
-//! | Extrinsic                                     | Issuer | Retail Investor | Professional Investor | Institutional Investor |
-//! |-----------------------------------------------|--------|-----------------|-----------------------|------------------------|
-//! | `create`                                      | X      |                 |                       |                        |
-//! | `edit_metadata`                               | X      |                 |                       |                        |
-//! | `start_evaluation`                            | X      |                 |                       |                        |
-//! | `start_auction`                               | X      |                 |                       |                        |
-//! | `bond_evaluation`                             |        | X               | X                     | X                      |
-//! | `failed_evaluation_unbond_for`                |        | X               | X                     | X                      |
-//! | `bid`                                         |        |                 | X                     | X                      |
-//! | `contribute`                                  |        | X               | X*                    | X*                     |
-//! | `vested_plmc_bid_unbond_for`                  |        |                 | X                     | X                      |
-//! | `vested_plmc_purchase_unbond_for`             |        | X               | X                     | X                      |
-//! | `vested_contribution_token_bid_mint_for`      |        |                 | X                     | X                      |
-//! | `vested_contribution_token_purchase_mint_for` |        | X               | X                     | X                      |
-//!
-//! _* They can call contribute only if the project is on the remainder round._
-//!
+//! 1) **Project Creation**: Issuer creates a project with the [`create_project`](Pallet::create_project) extrinsic.
+//! 2) **Evaluation Start**: Issuer starts the evaluation round with the [`start_evaluation`](Pallet::start_evaluation) extrinsic.
+//! 3) **Evaluate**: Evaluators bond PLMC on the project with the [`bond_evaluation`](Pallet::evaluate) extrinsic.
```

Comment:
Evaluators bond PLMC to evaluate a project with the [`bond_evaluation`](Pallet::evaluate) extrinsic.